samples/esb/README.rst

```diff
@@ -19,10 +19,10 @@ Successful communication is indicated by LED changes, which should be in sync on
 Transmitter
 ===========
 
-The Transmitter example sends a packet, waits for a configurable time (50 milliseconds by default), and then sends another packet.
+The Transmitter example sends a packet, waits for a configurable time (50-milliseconds by default), and then sends another packet.
 In each packet, the four least significant bits of the first byte of the payload are incremented (or reset to zero when they reach 16).
 The LEDs indicate that packets were sent and acknowledged.
-Therefore, if packets are successfully received and acknowledged by the Receiver, the LED pattern will change every 50 milliseconds (with the default delay).
+Therefore, if packets are successfully received and acknowledged by the Receiver, the LED pattern will change every 50-milliseconds (with the default delay).
```

Comment:
Sorry for pointing this out only now, but I think hyphens are not needed here. They would be needed for example in case of "will change with the 50-millisecond default delay", but here "50 ms" are not used as modifiers to a noun to justify a hyphen.

Reply:
Ok.